server/graphql_schemas/attend/schema.py

```diff
@@ -76,6 +77,12 @@ def resolve_attenders_list(self, info, **kwargs):
             Q(event__creator__user=info.context.user)
         )
 
+    def resolve_attending_list(self, info, **kwargs):
+        return Attend.objects.filter(
```

Comment:
The query here makes sense, but we can simply have
```
return Attend.objects.filter(user__user=info.context.user, status='attending')
```
Since keyword argument queries in filter() are “AND”ed together.

I'd like to know if you think this is okay.

Reply:
It's fine. Since it's an `AND` filter and `Q` filters normally help with `OR` filters.

Reply:
Exactly! Thanks 👍